src/main.py

```diff
@@ -226,8 +226,7 @@ async def task_add_unupdated_role():
         await member.remove_roles(unupdated_role)
 
     print("done")
-
-@tasks.loop(time=time(hour=23, minute=45))
+@tasks.loop(time=datetime.time(hour=23, minute=45))
```

Comment:
oops I meant 2 newlines instead of 0 😭 that's the standard above/below a function in black